www/js/calendar.js

```diff
@@ -1,19 +1,18 @@
 // Init calendar if not already inited
-$$(document).on('page:init', '.page[data-name="calendar"]', function (e) {
-  var page = $('.page.calendar-page');
+$$(document).on('page:init', '.page[data-name="calendar"]', function () {
+  const page = $('.page.calendar-page');
```

Comment:
@fsek/devs Only very old android devices don't support const/let now, and we can't run the app on those devices anyway. So there's no reason to keep using var anymore :)

Reply:
We should add that to eslint